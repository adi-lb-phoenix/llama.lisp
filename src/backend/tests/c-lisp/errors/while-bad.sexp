(c-lisp
    (define ((main int))
        (while
            (call print 5))))
