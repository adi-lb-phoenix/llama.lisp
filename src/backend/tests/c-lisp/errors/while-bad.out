Error in statement: ['while', ['call', 'print', 5]]
CodegenError: Bad while statement: ['while', ['call', 'print', 5]]
